CHANGELOG.md

```diff
@@ -2,36 +2,66 @@
 
 ##### Changes
 
```

Comment:
Another thing I remembered related to the release process. It's missing the [Github releases](https://github.com/remoteoss/json-schema-form/blob/main/CONTRIBUTING.md#publishing-a-stable-release)

Reply:
I'll do that straight after merging this and include the changes there!